imperial.test.js

```diff
@@ -0,0 +1,113 @@
+import Imperial from "./imperial";
+import { Nation } from "./constants.js";
+
+describe("Imperial", () => {
+  describe("constructor", () => {
+    describe("six players", () => {
```

Comment:
Could this be straightforwardly done for 5, 4, 3, and 2 player games too? Is it worth writing out tests for those too?

Reply:
Yes, I am doing the 5, 4, 3 and 2 cases presently!